### ISSUE TITLE
Emojify `cabal check`

### ISSUE THREAD

**Issue by ffaf1** [haskell/cabal#8993]
From this [good suggestion](https://github.com/haskell/cabal/issues/8880#issuecomment-1572397131).

> @ffaf1 for the record, i think Cabal's messaging is severely under-developed in terms of decorations. Please, give me more emojis and alike! One of the ways to figure how to do it is to steal from someone, i think. E.g. cargo would be my first choice to look for an example. I guess, I could look into what Julia and its package manager do in such cases. I just need to figure out what would be the closest scenario…

A good article on accessibility considerations [here](https://uxcontent.com/accessible-content-design-for-emojis/).

**Comment by andreabedini** [comment]
I am not against experimenting with empjis but I would start from adding colours first.

If we start displaying info/warn/error messages in different colours, mistakes in this categorisation would become more obvious; and fixing those miscategorisation would improve everybody's experience, whether they see the colours or not.

Just my 2c ☺️

**Comment by ulysses4ever** [comment]
Yeah, I didn't mean emoji literally, tbh. More like 1) Unicode niceties in general, and 2) Colors.

**Comment by jneira** [comment]
please, take in mind fallback to simle shells without color or unicode support
